src/predicates/propEq.spec.js

```diff
@@ -0,0 +1,52 @@
+const test = require('tape')
+const helpers = require('../test/helpers')
+
+const bindFunc = helpers.bindFunc
+
+const isFunction = require('../core/isFunction')
+const unit = require('../core/_unit')
+
+const propEq = require('./propEq')
+
+test('propEq function', t => {
+  const f = bindFunc(propEq)
+
+  t.ok(isFunction(propEq), 'is a function')
+
+  const err1 = /propEq: Number or String required for first argument/
```

Comment:
Would be nice to have these declared above the grouping they are being checked against. Makes it easier when adjusting specs in the future.

Reply:
Sure, makes sense. Apologies, I'm still getting used to the stylistic changes. I'll be cleaner the next time around :-)